rust/stracciatella_c_api/src/c/librarydb.rs

```diff
@@ -79,23 +72,18 @@ pub extern "C" fn LibraryFile_close(file: *mut LibraryFile) {
 /// Sets the rust error.
 #[no_mangle]
 pub extern "C" fn LibraryFile_seek(file: *mut LibraryFile, distance: i64, from: c_int) -> bool {
+    forget_rust_error();
     let file = unsafe_mut(file);
     let seek_result = match from {
         0 if distance >= 0 => file.seek(SeekFrom::Start(distance as u64)),
         1 => file.seek(SeekFrom::End(distance)),
         2 => file.seek(SeekFrom::Current(distance)),
         _ => Err(io::ErrorKind::InvalidInput.into()),
     };
-    match seek_result {
-        Err(err) => {
-            remember_rust_error(format!("{:?}", err));
-            false
-        }
-        Ok(_) => {
-            forget_rust_error();
-            true
-        }
+    if let Err(err) = seek_result {
+        remember_rust_error(format!("{:?}", err));
```

Comment:
Might be good to include library file name in the error message.

Reply:
Done